iac-extensions/terraform/src/test/java/org/sonar/iac/terraform/checks/azure/CertificateBasedAuthenticationCheckTest.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I saw, that you've created an empty line above each Java file. I already removed them.